packages/smooth_app/lib/generic_lib/bottom_sheets/smooth_draggable_bottom_sheet.dart

```diff
@@ -74,7 +74,7 @@ class SmoothDraggableBottomSheetState
                     child: Material(
                       type: MaterialType.transparency,
                       child: ClipRRect(
-                        borderRadius: widget.borderRadius,
+                        // borderRadius: widget.borderRadius,
```

Comment:
Please put it back. If it's a parameter we introduced, let's assume it is useful.

Reply:
type 'BorderRadius?' can't be assigned to the parameter type 'BorderRadiusGeometry'
What to do with this error then?

Reply:
In that case, as @g123k recently switched to flutter 3.13 (from 3.10), I guess it would make sense to "Update branch". That would probably include the fix to this error.
For the record: when there's an error, the trick is not to comment it, especially if your code gets reviewed :wink: